owncloud/core
# [stable9.1] Faster classification migration

fixes https://github.com/owncloud/core/issues/25617
- [ ] apply same mechanism for birthday sync @deepdiver1975

@sbraz mind testing - thx
## Test Scenario
1. Run occ upgrade
2. see the progress on processed calendar objects

```
deepdiver@alien:~/Development/ownCloud/master [faster-classification-migration]$ ./occ up
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Set log level to debug
Turned on maintenance mode
Checking whether the database schema can be updated (this can take a long time depending on the database size)
Checked database schema update
Checking updates of apps
Checking whether the database schema for <dav> can be updated (this can take a long time depending on the database size)
 Done                                                        
 7/7 [============================] 100%                     
Checked database schema update for apps
Updating database schema
Updated database
Updating <dav> ...
Fix classification for calendar objects
 Done                                                        
 6/6 [============================] 100%                     
Updated <dav> to 0.2.7
Drop old database tables
 Done                                                        
 28/28 [============================] 100%                   
Remove old (< 9.0) calendar/contact shares
 Done                                                        
 4/4 [============================] 100%                     
Fix permissions so avatars can be stored again
 Done                                                        
 2/2 [============================] 100%                     
Update successful
Turned off maintenance mode
Reset log level
```
